--- update_patches.sh.1
+++ update_patches.sh
@@ -19,3 +19,13 @@
 
-	echo Need to patch $last_file to make $stage
+	# Skip parsing of diff files
+	if [[ $stage == *.diff ]]; then
+	    continue
+	fi
+
+	echo Creating $stage.diff
+
+	# -Z Ignores space
+	# -U 1 sets to the unified patch format with one surrounding lines
+	# Setting --label directly lets use avoid timestamps
+	diff -Z -U 1 --label $last_file --label $stage $last_file $stage > $stage.diff
 
